hack/python-sdk/test-sdk.sh

```diff
@@ -15,30 +15,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# We always need to install python3-venv, so makes sense to update once
-apt update -y
-
-if [ -z `which python3` ]; then
-    # Default python for bookworm (golang 1.19/1.20) is 3.11
-    # so this should already be installed
-    apt-get install python3 -y
-fi
-
-if [ -z `which pip` ]; then
-    apt-get install python3-pip -y
-fi
-
 repo_root="$(dirname "${BASH_SOURCE}")/../.."
 
 cd "${repo_root}/sdk/python"
 
-# We need to create a virtual environment for testing
-apt-get install -y python3-venv
-python3 -m venv env
-source env/bin/activate
-
-# install test requirements
-pip install -r test-requirements.txt
+docker buildx build -f Dockerfile -t python-unit .
```

Comment:
It looks like the ci doesn’t detect docker..  any ideas on how to proceed?  I could switch to using the same runner we use for our e2e tests.

Reply:
Can we install docker in this script?

Reply:
A goal of this was to use docker so I could build on my dev environment (where I use yum).  I realize this is kinda of an opinioned PR but im trying to reproduce depencies on any OS.